style/themes/high-contrast.css

```diff
@@ -370,7 +370,7 @@ select:-webkit-autofill {
 .small-box > .small-box-footer {
   color: #fff;
 }
-.small-box > .small-box-footer {
+.small-box > .small-box-footer:hover {
   background: rgba(0, 0, 0, 0.3);
 }
```

Comment:
No. The darker background should be used **without** `hover`.

I don't remember what I did, but it should be just one selector containing both rules:

```css
.small-box > .small-box-footer {
  color: #fff;
  background: rgba(0, 0, 0, 0.3);
}
```